File: QE selenium java/Activity7.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity7 {

    public static void main(String[] args) throws InterruptedException {

        WebDriver driver = new FirefoxDriver();
        driver.manage().window().maximize();

        driver.get("https://training-support.net/webelements/dynamic-controls");

        System.out.println("Page Title: " + driver.getTitle());

        WebElement textField = driver.findElement(By.id("input-text"));

        System.out.println("Text field enabled before clicking Enable: "
                + textField.isEnabled());

        WebElement enableButton = driver.findElement(By.id("toggleInput"));
        enableButton.click();

        Thread.sleep(3000);

        System.out.println("Text field enabled after clicking Enable: "
                + textField.isEnabled());

        driver.quit();
    }
}

File: QE-selenium python/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

driver = webdriver.Firefox()
driver.maximize_window()

driver.get("https://training-support.net/webelements/dynamic-controls")

print("Page Title:", driver.title)

text_field = driver.find_element(By.ID, "input-text")

print("Text field enabled before clicking Enable:",
      text_field.is_enabled())

enable_button = driver.find_element(By.ID, "toggleInput")
enable_button.click()

time.sleep(3)

print("Text field enabled after clicking Enable:",
      text_field.is_enabled())

driver.quit()
